src/borg/helpers/passphrase.py

```diff
@@ -111,19 +115,27 @@ def verification(cls, passphrase):
         ):
             print('Your passphrase (between double-quotes): "%s"' % passphrase, file=sys.stderr)
             print("Make sure the passphrase displayed above is exactly what you wanted.", file=sys.stderr)
+            print(
+                "Your passphrase (UTF-8 encoding in hex): %s" % bin_to_hex(passphrase.encode("utf-8")), file=sys.stderr
+            )
             try:
                 passphrase.encode("ascii")
             except UnicodeEncodeError:
-                print(
-                    "Your passphrase (UTF-8 encoding in hex): %s" % bin_to_hex(passphrase.encode("utf-8")),
-                    file=sys.stderr,
-                )
                 print(
                     "As you have a non-ASCII passphrase, it is recommended to keep the "
                     "UTF-8 encoding in hex together with the passphrase at a safe place.",
                     file=sys.stderr,
                 )
```

Comment:
guess we can always do that print. as i explained in the ticket, there can be issues even with ascii passphrases.

"It is recommended to keep the UTF-8 encoding in hex together with the passphrase at a safe place. In case you should ever run into passphrase issues, it can sometimes help debugging them."